client/src/components/ListingEntry.jsx

```diff
@@ -51,12 +51,12 @@ class ListingEntry extends React.Component {
               <ReviewCount>{`(${this.props.listing.reviewCount})`}</ReviewCount>
             </Rating>
             <RoomDescription>
-              <Text>{`${this.props.listing.roomType} · `}</Text>
+              <Text>{`${this.props.listing.room_type} · `}</Text>
```

Comment:
Only comment here is trying to figure out if room and property are the same thing here? For consistency it might help to choose one or the other throughout. Or am I misunderstanding what "room" references here?

Reply:
I chose a different nomenclature for db architecture than Andrew V did in the client view who wrote the app originally. It is too much work to change every reference in the client app for this project, as there are higher priority tasks that need to be taken care of. 

For now, properties and rooms are the same.

Reply:
Makes sense.